### ISSUE TITLE
Create yesod container

### ISSUE THREAD

**Issue by phrostbyten** [etz4ai/yesod#1]
Try to make the bare minimum needed to run MKL

**Comment by rashley-iqt** [comment]
you mean the Math Kernel Library correct? And if so, would creating a container that has TensorFlow compiled to use mkl-dnn work?

**Comment by phrostbyten** [comment]
That's right Intel Math Kernel Library. TensorFlow probably belongs in the yesod-python container that would build on top of this one (using a FROM directive).
